ui/packages/catalog-ui-search/src/main/webapp/js/properties.js

```diff
@@ -172,6 +172,9 @@ const properties = {
   isMetacardPreviewEnabled() {
     return !this.isMetacardPreviewDisabled
   },
+  isValidationEnabled() {
+    return !this.isClientValidationEnabled
```

Comment:
I think we should keep the naming consistent and don't need to invert the value here
```suggestion
  isClientValidationEnabled() {
    return this.isClientValidationEnabled
```